sklearn/metrics/setup.py

```diff
@@ -19,6 +19,13 @@ def configuration(parent_package="", top_path=None):
         "_pairwise_fast", sources=["_pairwise_fast.pyx"], libraries=libraries
     )
 
+    config.add_extension(
+        "_pairwise_distances_reduction",
+        sources=["_pairwise_distances_reduction.pyx"],
+        language="c++",
```

Comment:
I could not spot C++ code (though I would love to).

Reply:
That's right, C++ is unneeded but previously was needed for `std::vector` introduced for querying using radius.

This will be introduced in a subsequent PR. :wink: